src/Owin.Security.OpenIdConnect.Server/OpenIdConnectServerHandler.Endpoints.cs

```diff
@@ -704,6 +704,37 @@ internal partial class OpenIdConnectServerHandler : AuthenticationHandler<OpenId
             }
         }
 
+        private bool TryGetClientCredentialsFromHeader(OpenIdConnectMessage request)
```

Comment:
Yeah, I was trying to reuse code without copying it all, but the usage is different between endpoints. I'll revert.